docs/Screenshots.stories.mdx

```diff
@@ -0,0 +1,55 @@
+<Meta
+  title="Guidelines/Screenshot testing"
+  parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+## Screenshot testing
+
+For the starting work with the screenshot test locally you need to start `storybook`:
+
+```bash
+$ npm run start
+```
+
+For the matching testing screenshots correctly, they should create in a similar OS (Ubuntu),
+to generate and update testing screenshots locally need to install the [docker](https://www.docker.com/) on the local machine and run the next command.
+
+```bash
+$ npm run test:sh:docker
+```
+
+For the updating screenshots images locally:
+
+```bash
+$ npm run test:sh:update
+```
+
+### Screenshot files
+
+Screenshot files are stored in [git-lfs](https://git-lfs.github.com/).
+
+Installing `git-lfs` on MacOS:
+
+```bash
+$ brew install git-lfs
+```
+
+**Then you need to install the `git hook`:**
+
+```bash
+git lfs install --force
+```
+
+Further, you can work with screenshot files in the git as with regular files.
```

Comment:
suggestion:
`Finally, you are able to work with screenshot files in git as well as with regular files.`